src/test/java/com/training/assignments/Assignment2Test.java

```diff
@@ -0,0 +1,25 @@
+package com.training.assignments;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class Assignment2Test {
+    Assignment2 assignment2 = new Assignment2();
+
+    @Test
+    public void testString1() {
+        Assertions.assertEquals("Sam", assignment2.getStringReplace("Ram", "Sam"));
```

Comment:
As all  3 methods are performing same operation can you please use one method instead of using 3 methods separately-->testStringReplace()